app/shop.py

```diff
@@ -0,0 +1,41 @@
+import datetime
+
+
+class Shop:
+    def __init__(
+            self,
+            name: str,
+            location: list[int],

```

Comment:
The use of `list[int]` for the `location` parameter is not ideal if you are expecting a pair of coordinates. Consider using a more descriptive type or a custom class to represent the location.